Se agregan diagramas de aeródromos MMID

enriquea88 opened the pull request:
MMCP, MMCM, MMCT, MMCE, MMCZ, MMIT, MMMT, MMPQ, MMTP, MMTG, MMVA

enriquea88 committed:
Se agregan diagramas de aeródromos MMID

MMCP, MMCM, MMCT, MMCE, MMCZ, MMIT, MMMT, MMPQ, MMTP, MMTG, MMVA

hatoriganzo commented on Maps/Campeche/ASMGCS_MMCP.xml:
```diff
@@ -0,0 +1,204 @@
++194852.099-0903013.897/
++194851.955-0903014.529</Point>
+    </Infill>
+
```
Remover espacios vacios

hatoriganzo commented on Maps/Campeche/ASMGCS_MMCP.xml:
```diff
@@ -0,0 +1,204 @@
++194855.285-0903009.887/
++194854.558-0903013.128</Point>
+    </Infill>
+ 
```
espacios

hatoriganzo commented on Maps/Chichen Itza/ASMGCS_MMCT.xml:
```diff
@@ -0,0 +1,138 @@
+  </Map>
+  <Map Type="Ground_RWY" Name="ASMGCS_MMCT_LABELS" CustomColourName="GroundLabel" Priority="1" Center="+203828.510-0882654.000">
+  
+   <Label HasLeader="false">
```
indentación

hatoriganzo commented on Maps/Ciudad del Carmen/ASMGCS_MMCE.xml:
```diff
@@ -0,0 +1,204 @@
+  <Map Type="Ground_RWY" Name="ASMGCS_MMCE_LABELS" CustomColourName="GroundLabel" Priority="1" Center="+183906.380-0914758.720">
+  
+   <Label HasLeader="false">
+		<Point Name="A">+183917.230-0914821.060</Point>
```
Indentación, recuerda que debe ser 2 espacios

enriquea88 committed:
Agregar diagramas aeródromos MMID y modifica Positions.xml

MMCP, MMCM, MMCT, MMCE, MMCZ, MMIT, MMMT, MMPQ, MMTP, MMTG, MMVA, Positions.xml

gvaldezsan commented:
@enriquea88 Te vuelvo a rechezar el Pull Request, es muy importante la consistencia en indentación y espacios. Te corregí el archivo ASMGCS_MMCP.xml. Con ese formato deben quedar todos porfa.

Saludos!

hatoriganzo committed:
Arregla indentación en MMCP

hatoriganzo committed:
Fixes last

enriquea88 committed:
Se corrige código aeródromos MMID

Indentación correcta aplicada

gvaldezsan commented:
@enriquea88 Rechazado, solamente falta correción de Positions.xml

enriquea88 committed:
Corrección Positions.xml

Se agrega espaciamiento correcto en el código y modifica orden de diagramas terrestres

hatoriganzo commented on Positions.xml:
```diff
@@ -59,7 +59,7 @@
       </Maps>
     </Position>
-             <Position Name="HOA - Aproximación Hermosillo" Type="ASD" DefaultCenter="+290544.93-1110252.29" DefaultRange="225" MagneticVariation="-9">
+             <Position Name="Aproximación Hermosillo - HOA" Type="ASD" DefaultCenter="+290544.93-1110252.29" DefaultRange="225" MagneticVariation="-9">
```
Remover y arreglar indentacion

hatoriganzo commented on Positions.xml:
```diff
@@ -107,7 +107,7 @@
       </Maps>
     </Position>
-    <Position Name="SDA - Aproximación San Jose" Type="ASD" DefaultCenter="+230846.179-1094317.540" DefaultRange="225" MagneticVariation="-8">
+    <Position Name="Aproximación San Jose - SDA" Type="ASD" DefaultCenter="+230846.179-1094317.540" DefaultRange="225" MagneticVariation="-8">
```
Remover y arreglar indentacion

hatoriganzo commented on Positions.xml:
```diff
@@ -155,7 +155,7 @@
       </Maps>
     </Position>
-             <Position Name="TJA - Aproximación Tijuana" Type="ASD" DefaultCenter="+323227.810-1165811.890" DefaultRange="225" MagneticVariation="-12">
+             <Position Name="Aproximación Tijuana - TJA" Type="ASD" DefaultCenter="+323227.810-1165811.890" DefaultRange="225" MagneticVariation="-12">
```
Remover y arreglar indentacion

hatoriganzo committed:
Arregla cambios en falso

hatoriganzo committed:
Fixes last

hatoriganzo committed:
Merge branch 'airac-2014' into diagramas-MMID

gvaldezsan commented:
Finalizada
